### PR TITLE
Full skill, augment, and tactics trial translations

### PR DESCRIPTION
**Pull request by KomradeSpectre**
Another swathe of translations for you. If you know how to prevent the old commits from showing up in the pull, please let me know because I feel pretty embarrassed that they're continuously there haha.

### CODE REVIEW

**Comment by riftcrystal** [comment]
@KomradeSpectre Could try `git pull --rebase; git push -f origin master`.

**Comment by KomradeSpectre** [comment]
It doesn't include the old commits in the files changed though, so it's fine to pull still (after checking for anything I may have done wrong). I'm just confused on where I should focus next to be honest. I'll see about fixing the commit history thing in the next pull though.

**Comment by KomradeSpectre** [comment]
Seems one or two tactics trials made their way under my nose. To my knowledge there was only 1 per class. I'll fix this issue by translating the ones I forgot, and will commit it here.

File: ui/00_message/quest/q00000002.xml [hunk 1]
```diff
@@ -3,90 +3,129 @@
     <original>【ヒント：新たなメインクエストについて】
 新たな&lt;COL ff7575&gt;メインクエスト＜和解の使者＞&lt;/COL&gt;が受注可能になりました
 謁見の間にいるレオに話しかけて受注しましょう</original>
+    <translation>【Tip: About New Main Quests】
```
Comment (riftcrystal) [inline ui/00_message/quest/q00000002.xml]: Would be good to move the 【】 over to [].

File: ui/00_message/skill/custom_skill_command_03.xml [hunk 2]
```diff
@@ -61,11 +61,12 @@
   </message>
   <message>
     <key>CUSTOM_SKILL_COMMAND_03_131</key>
-    <original>セットボタンを押す
-</original>
```
Comment (riftcrystal) [inline ui/00_message/skill/custom_skill_command_03.xml]: Don't edit original text. Even if it has formatting mistakes.

**Review by riftcrystal — CHANGES_REQUESTED** [review]

File: ui/00_message/skill/custom_skill_command_03.xml (hunk 2 above)
Comment (KomradeSpectre) [reply ui/00_message/skill/custom_skill_command_03.xml]: That was a mistake on my behalf, not intentional.

Comment (riftcrystal) [reply ui/00_message/skill/custom_skill_command_03.xml]: That's fine, just pointing it out. My tooling requires original text to be left alone for things like merging in patch changes, etc.

Comment (KomradeSpectre) [reply ui/00_message/skill/custom_skill_command_03.xml]: I'll fix it in a moment.

**Comment by KomradeSpectre** [comment]
I think that should be good? Lemme know if you catch anything else. I try to learn from the mistakes I make on here haha

**Comment by riftcrystal** [comment]
There will probably be necessary formatting changes to be done (wrapping, et al), but looks good.

**Pull request merged**